resolwe/flow/utils/purge.py

```diff
@@ -164,6 +164,24 @@ def _location_purge_all(delete=False, verbosity=0):
         logger.info("No data locations")
 
 
+def storage_purge(delete=False, verbosity=0):
```

Comment:
It is not intuitive that `location_purge` purges single location and `storage_purge` purges all storages.
I don't see a use-case where this function would be used separately. So you can just rename it to `_storage_purge_all` and only use it in `purge_all`.